documentation/java.asciidoc

```diff
@@ -49,6 +67,20 @@ devon java create com.example.domain.myapp -DdbType=oracle -Dversion=0.0.1 com.e
 ```
 Will create an app with package `com.example.domain.myapp`, groupId `com.example.group`, artifactId `myapp`, version `0.0.1`, oracle database, and with a batch module.
 
+The -f parameter can be added to every command.
+
+=== Quarkus 
+Examples for create a new devon4j Quarkus application:
```

Comment:
I am not sure if we should call it 'devon4j Quarkus application', since it is actually a normal Quarkus application

Reply:
I have decided to call that way, because I also create the devon project structure, so the quarkus projekt has changed slightly. I am also trying to create an archtype then it would be correct to call it devon4j quarkus application, right?

Reply:
Yeah, I think then it's okay to call it that way